### DISCUSSION TITLE
發展人工智能是福是禍

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#217] [General]
人工智能（AI）作為近年來的一門新興技術，已經悄然地應用到人們生活的方方面面。人工智能正在引領着一場新的技術革命，為人類的生活帶來了很多便捷，亦為人類創造出了非常可觀的經濟效益。然而，發展如此迅速的人工智能，既有利，也有弊。
https://ls.chiculture.org.hk/tc/hot-topics/656

**Comment by handsomeniko** [comment]
我這學期有修AI倫理導論，其實裡面就有探討到類似主題，有興趣可以去修~

**Comment by Shiba010** [comment]
我認爲AI對人類絕對是利大於弊。

**Comment by yasklop** [comment]
AI真是好處多多

**Comment by ghost** [comment]
AI絕對是各有利弊 但倒底一定是利大於弊 不然就不會發展AI了

**Comment by lx3211** [comment]
有許多人因此失業，但也因此產生了一些新職業，究竟是福是禍可能要看身處的角度了吧

**Comment by brian431** [comment]
無論如何人類的科技都是必須要持續地發展的，至於所帶來的負面影響就只能逐漸解決

**Comment by NcuJerryKao** [comment]
若發展AI 人類的科技將會大躍進 以更誇張的速度進步
至於AI將會帶來的危機 感覺應該控制好就沒事了吧 訂定三大協議，糟糕要提到電影了

**Comment by BingRong1217** [comment]
每件事物都是一體兩面的

**Comment by Yilin20021016** [comment]
所有東西都有不同的優缺點 只是從那個角度看

**Comment by larry40305** [comment]
儘管如此，我們仍然需要對AI發展保持懷疑的態度。

**Comment by HsuChen88** [comment]
討論來討論去，我們也不會因為他有利或有弊而停止發展。
